Tests/Test/ResponseTest.swift

```diff
@@ -7,59 +7,60 @@ class ResponseTest: XCTestCase {
     let response = Response(parser: parser)
     static let body200 = "<!DOCTYPE html><html><body><h1>Hello World</h1></body></html>"
     static let body404 = "<!DOCTYPE html><html><body><h1>404 Page Not Found</h1></body></html>"
+    let requestHeader = "\r\nCache-Control: no-cache\r\nConnection: keep-alive\r\n\r\n"
```

Comment:
Think about abstracting `\r\n` to a constant. With this very specific & important pattern 'magic stringed' all over the place you're leaving yourself very vulnerable to typos.